glommio/src/executor/mod.rs

```diff
@@ -4148,6 +4152,40 @@ mod test {
             .unwrap();
     }
 
+    #[test]
+    fn blocking_function_placement_independent_of_executor_placement() {
+        let num_schedulable_cpus = LocalExecutorBuilder::new(Placement::Fixed(0))
+            .blocking_thread_pool_placement(PoolPlacement::Unbound(4))
```

Comment:
I am afraid this test will break when executed in machines with less than 4 CPUs.

Should we skip it in that scenario? It's still a valuable test

Reply:
This doesn't actually need 4 CPUs. Can be reduced to 2 and I'll adjust to skip the test if there's only 1 online CPU found.

Reply:
that'd be better